packages/rehype-minify-whitespace/index.js

```diff
@@ -116,7 +116,8 @@ function collapsable(node) {
 /* Collapse to spaces, or newlines if they’re in a run. */
 function collapseToNewLines(value) {
   var result = String(value).replace(/\s+/g, function($0) {
-    return $0.indexOf('\n') === -1 ? ' ' : '\n'
+    const newLinesCount = ($0.match(/\n/g) || []).length
```

Comment:
Could you add some comments why 2 newlines are added?

Reply:
It's a leftover from when I was exploring some solutions.
I'll check but it seems unneeded.